devopera.webmanifest

```diff
@@ -3,8 +3,8 @@
 	"name": "Dev.Opera",
 	"icons": [
 		{
-			"src": "apple-touch-icon-144x144.png",
-			"sizes": "144x144",
+			"src": "touch-icon.png",
```

Comment:
> Let’s keep the apple-touch-icon.png name and reference that here.

@mathiasbynens The `apple-touch-icon.png` right now is only `180×180px`, while the recommended size for Android is [`192×192px`](https://developer.chrome.com/multidevice/android/installtohomescreen#best-practices). 

Do you want me to just use one `192×192px` image named `apple-touch-icon.png`?